src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -659,10 +659,19 @@ export interface ITerminalInstance {
 	registerMarker(): IMarker | undefined;
 
 	/**
-	 * Adds a decoration to the buffer at the @param marker with
-	 * @param genericMarkProperties
+	 * Adds a marker to the buffer, mapping it to an ID if provided.
+	 * When not @param hidden, adds a decoration to the buffer
```

Comment:
`@param` part should be removed as hidden no longer exists. Optional to add `@param properties` is it seems self explanatory.